docs/testing.md

```diff
@@ -247,51 +247,86 @@ Jest and React Testing Library are frequently used together for **Unit Testing**
 
 ### Quickstart
```

Comment:
I wonder if I'm being pedantic here, what are your thoughts on converting this into steps, using the same heading titles?

Reply:
Could you expand on what you mean here? 😊

Reply:
Sorry. I just mean, would It be clearer for us to explain the QuickStart as a set of steps:

```
## Step 1: install

## Step2: do something else
```

This may not be the case here. I don't have a hard and fast rule on this, I just think it makes skimming content easier, you can drop in a a step if need etc.. I don't think this fits in all cases, was wondering what your thoughts are with this section.

I quite like how you have done it tbh, just raising the question. ☺️

Reply:
Ah, thank you! I agree with you, it's good to give people context at the start. As you said, I'm not sure how the steps would fit this case, but I've added a [quick summary here](https://github.com/vercel/next.js/pull/31633/commits/3968e9afe0e2a5d3a22d1ad42d013224f12deca6). wdyt?